### ISSUE TITLE
More ideologies(primitivism,gender etc)

### ISSUE THREAD

**Issue by pokeball99** [8values/8values.github.io#27]
Great work your doing, but at this point it still got quite a bit to go for an encompassing test, say for example the UNAbomber wouldn't have his political views correctly(he's an Anarcho-primitivism)
https://en.wikipedia.org/wiki/List_of_political_ideologies might help

**Comment by Saphareas** [comment]
AFAIK, this is already in the making and a major update will arrive soon.

**Comment by TristanBomb** [comment]
Yeah, I'm currently trying to rethink ideology matching. There's still a ways to go, though!
